test/e2e/components/vl-functional-header.js

```diff
@@ -0,0 +1,23 @@
+const { VlElement } = require('vl-ui-core').Test;
+const { By } = require('vl-ui-core').Test.Setup;
+const { VlLink } = require('vl-ui-link').Test.VlLink;
+
+class VlFunctionalHeader extends VlElement {
+    async getTitle() {
+        const element = await this.shadowRoot.findElement(By.css('#title'));
+        return new VlElement(this.driver, element);
+    }
+
+    async getSubTitle() {
+        const element = await this.shadowRoot.findElement(By.css('#sub-title'));
+        return new VlElement(this.driver, element);
+    }
+
```

Comment:
moet je slot niet kunnen opvragen van deze?

Reply:
Leek me niet nodig, hier leun ik op de `getText` implementatie op vl-ui-core niveau die gaat kijken of het element een slot is of niet om te bepalen hoe de tekst opgevraagd moet worden.